terracumber-cli

```diff
@@ -20,6 +20,9 @@ def parse_args():
     """Parse arguments"""
     parser = argparse.ArgumentParser(description='Run terrafrom and cucumber')
     parser.add_argument('--tf', help='Path to main.tf file', required=True)
+    parser.add_argument('--tfvars_files', action='append', help='Path to tfvars file', required=False,
+                        default=[])
+    parser.add_argument('--tfvariables_file', help='Path to variable.tf file', required=False, default='')
```

Comment:
Please provide a better description and a name that can identify them better.
They look too close and it's hard for someone reading the code for first time to distinguish them.